src/util/animator.js

```diff
@@ -42,6 +42,59 @@ var RectangleElement = function(id, width, height, color, stroke_color,
   };
 };
 
+// An element that is a colored circle.
+var CircleElement = function(id, radius, color, stroke_color,
+                             line_width) {
+    var circle = this;
```

Comment:
Can you change the indentation to two spaces, to standardize with the other files?